### PR TITLE
Add UT to RBD volume test of TestGetAccessModes and TestRequiresRemount

### PR DESCRIPTION
**Pull request by jianglingxia**
**What this PR does / why we need it**:
Add UT to RBD volume test of TestGetAccessModes and TestRequiresRemount
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Review by xiaoxubeii — CHANGES_REQUESTED** [review]
I think your codes should match the pr title.

**Comment by jianglingxia** [comment]
thx,I have fixed it! @xiaoxubeii

**Comment by xiaoxubeii** [comment]
The pr commit msg should also be changed.

**Comment by jianglingxia** [comment]
very thanks! @xiaoxubeii 
for review @jsafrane thanks!

File: pkg/volume/rbd/rbd_test.go [hunk 1]
```diff
@@ -649,3 +649,33 @@ func TestConstructVolumeSpec(t *testing.T) {
...
+	}
+}
+
+func TestRequiresRemount(t *testing.T) {
+	tmpDir, _ := utiltesting.MkTmpdir("rbd_test")
+	plugMgr := volume.VolumePluginMgr{}
+	plugMgr.InitPlugins(ProbeVolumePlugins(), nil /* prober */, volumetest.NewFakeVolumeHost(tmpDir, nil, nil))
+	plug, _ := plugMgr.FindPluginByName("kubernetes.io/rbd")
+	has := plug.RequiresRemount()
+	if !has {
```
Comment (jsafrane) [inline pkg/volume/rbd/rbd_test.go]: I am missing some `t.Error` here, the test always passes.

Comment (jianglingxia) [reply pkg/volume/rbd/rbd_test.go]: you mean that the return change to t.Error()? i recommit it ,can you review it again?thanks

File: pkg/volume/rbd/rbd_test.go [hunk 2]
```diff
@@ -649,3 +649,33 @@ func TestConstructVolumeSpec(t *testing.T) {
...
+}
+
+func TestRequiresRemount(t *testing.T) {
+	tmpDir, _ := utiltesting.MkTmpdir("rbd_test")
+	plugMgr := volume.VolumePluginMgr{}
+	plugMgr.InitPlugins(ProbeVolumePlugins(), nil /* prober */, volumetest.NewFakeVolumeHost(tmpDir, nil, nil))
+	plug, _ := plugMgr.FindPluginByName("kubernetes.io/rbd")
+	has := plug.RequiresRemount()
+	if has {
+		t.Errorf("the plugin no need RequiresRemount")
```
[the remark this answers is not in the document]
Comment (jianglingxia) [reply pkg/volume/rbd/rbd_test.go]: for your review,thanks

**Pull request merged**